packages/admin-next/dashboard/src/routes/products/product-create/components/product-create-form/product-create-form.tsx

```diff
@@ -105,12 +107,12 @@ export const ProductCreateForm = () => {
 
         handleSuccess(`../${product.id}`)
       } catch (error) {
-        if (isFetchError(error) && error.status === 400) {
-          toast.error(t("general.error"), {
-            description: error.message,
-            dismissLabel: t("general.close"),
-          })
-        }
+        // if (isFetchError(error) && error.status === 400) {
+        toast.error(t("general.error"), {
```

Comment:
Suggestion: could we revert that, add a return in the if and add the following after the id
```ts
          toast.error(t("general.error"), {
            description: t("errors.serverError"),
            dismissLabel: t("general.close"),
          })
        }
```